### ISSUE TITLE
Docstrings missing for constant values in C++ headers.

### ISSUE THREAD

**Issue by achary** [rust-lang/rust-bindgen#1995]
### Input C/C++ Header

```C++
// test.hpp

#pragma once

/// This is a constant that has a docstring
///
/// And expected be be found in generated bindings code too.
const int FOO = 1;

/// This is a constant that has a docstring
///
/// And expected be be found in generated bindings code too.
struct Bar
{
    int baz;
};
```

### Bindgen Invocation

```Rust
// in build.rs 

    let bindings = bindgen::Builder::default()
        .header("test.hpp")
        .layout_tests(false)
        .generate_comments(true)
        .generate()
        .expect("Unable to generate bindings");

    bindings
        .write_to_file(out_path.join("test.rs"))
        .expect("Couldn't write bindings!");
```

### Actual Results

```rust
/* automatically generated by rust-bindgen 0.57.0 */

pub const FOO: ::std::os::raw::c_int = 1;
#[doc = " This is a constant that has a docstring"]
#[doc = ""]
#[doc = " And expected be be found in generated bindings code too."]
#[repr(C)]
#[derive(Debug, Copy, Clone)]
pub struct Bar {
    pub baz: ::std::os::raw::c_int,
}

```

### Expected Results

Markdown docstring for the const `FOO` should be present in the output `*.rs` file as well.

**Comment by emilio** [comment]
`impl CodeGenerator for Var` doesn't look at `item.comment()`, and it probably should, should be trivial to fix :)

**Comment by emilio** [comment]
this was fixed in #2027, thanks!
